src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -2018,83 +2017,71 @@ namespace nupic {
     }
     NTA_CHECK(check_vector_eq(trueNeighborsMap5, neighborsMap));
 
+    dimensions.clear();
+    dimensions.push_back(8);
+    numColumns = 8;
 
```

Comment:
Tidied up this unit test. It now has more consistent formatting. The larger (30 column) 2D tests could be separated out still.